### PR TITLE
modify reducer for counter

### PR DESCRIPTION
**Pull request by fncischen**
@ryanboris - begun assignment after long review of documentation.

### CODE REVIEW

File: src/reducers/index.js [hunk 1]
```diff
@@ -9,11 +9,19 @@ const initialState = {
...
 export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
+      return Object.assign({}, state, {
+        count: state.count + 1
+      })
     // Fill in the body of this case
-    case DECREMENT:
+    case DECREMENT: 
+      return Object.assign({}, state, {
```
Comment (secantsquared) [inline src/reducers/index.js:22]:
```suggestion
      return Object.assign({}, state, {
      not incorrect, but you might want to try using the spread operator to shallow copy its fun I promise! :)
```

**Review by secantsquared — COMMENTED** [review]
Good start and great structure and set up of the code.

File: src/reducers/index.js (hunk 1 above)
Comment (fncischen) [reply src/reducers/index.js:22]: will check it out!

**Comment by fncischen** [comment]
@ryanboris I've added a visual diagram of my understanding of Redux here. 👍

**Pull request merged**